### ISSUE TITLE
TypeError: h.slice(...).sort(...).flatMap is not a function

### ISSUE THREAD

**Issue by tiffanyhan** [ParabolInc/parabol#4706]
https://sentry.io/organizations/parabol/issues/2236601365/?referrer=github_plugin

```
TypeError: h.slice(...).sort(...).flatMap is not a function
  at a (./packages/client/components/RetroReflectPhase/PhaseItemColumn.tsx:185:8)
  at c (./packages/client/components/RetroReflectPhase/PhaseItemColumn.tsx:181:23)
  at Set.forEach (<anonymous>)
...
(29 additional frame(s) were not displayed)
```

**Comment by tiffanyhan** [comment]
first surfaced in release 6.0.0

**Comment by mattkrick** [comment]
closed in #4751
